boards/tutorials/nrf52840dk-thread-tutorial/src/main.rs

```diff
@@ -175,25 +205,34 @@ pub unsafe fn main() {
     let process_management_capability =
         create_capability!(capabilities::ProcessManagementCapability);
 
-    kernel::process::load_processes(
-        board_kernel,
-        chip,
-        core::slice::from_raw_parts(
-            core::ptr::addr_of!(_sapps),
-            core::ptr::addr_of!(_eapps) as usize - core::ptr::addr_of!(_sapps) as usize,
-        ),
-        core::slice::from_raw_parts_mut(
-            core::ptr::addr_of_mut!(_sappmem),
-            core::ptr::addr_of!(_eappmem) as usize - core::ptr::addr_of!(_sappmem) as usize,
-        ),
-        &mut *addr_of_mut!(PROCESSES),
-        &FAULT_RESPONSE,
-        &process_management_capability,
-    )
-    .unwrap_or_else(|err| {
-        debug!("Error loading processes!");
-        debug!("{:?}", err);
-    });
+    let loader = static_init!(
```

Comment:
We have a component for this, can it be used?

https://github.com/tock/tock/blob/master/boards/configurations/nrf52840dk/nrf52840dk-test-appid-sha256/src/main.rs#L290

Reply:
Now using this component https://github.com/tock/tock/pull/4109/commits/6307c2ce06632fdc60228428f8e040098eae79f8